bedrock2/Makefile.coqflags

```diff
@@ -0,0 +1,4 @@
+.PHONY: all print_coqc
+-include Makefile.coq.ex.conf
+print_coqc:
+	@printf "%s %s\n" "$(COQMF_COQLIBS)" "$(COQMF_OTHERFLAGS)"
```

Comment:
When does this file come into play?

Reply:
I use it to run `bytedump.sh` from the command line.

Reply:
Does the Makefile or any checked-in file contain a command that depends on this `Makefile.coqflags`? If not, it would be good to add a comment as to why this file is here.